1apply-css/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>
+      Terri's Examples: Applying CSS to HTML With Cascade and Specificity
+    </title>
+    <link rel="stylesheet" href="1styles.css" />
+  </head>
+
+  <body>
+<div class="main-container">
+    <h1><strong>Terri's Example: Applying CSS to HTML With Cascade and Specificity</strong></h1>
+    <p class="main__paragraph">
+      My cat Tina is a <strong style="color: orange">pain</strong> when it comes
+      to wanting her breakfast first thing in the morning!
+    </p>
+      <div class="main__paragraph"> <strong class="highlight">Finny</strong>, our other cat, will wait
+      patiently without meowing and <strong>scratching</strong> at the door.</div>
+     
+      <figure class="main__image">
+        <img src="images/shark.jpg"
+     alt="Cartoon depicting Sammy the Shark"
+     title="Sammy the Shark" />
+    <figcaption>The original example in our lesson featured "Sammy the Shark!"</figcaption>
+</figure>
+    </div>
+    <h2 class="main__title">What is demonstrated on this page:</h2>
+    <div class="main__list">
+    <ul class="main__ul">
+      <li class="main__li">
+        the bold text is red because the "strong" style in the styles.css has a
+        red color and I gave it an !important flag. That over-rides everything!
+      </li>
+
+      <li class="main__li">
+        If you comment out the strong style on line 12 of styles.css, then you
+        get green text for the "Finny" text because it takes the "highlight"
+        class color of green.
+      </li>
+
+      <li class="main__li">
+        "Pain" would then be orange because it has an inline style of orange for
+        the strong tag.
+      </li>
+
+      <li class="main__li">
+        "Scratching" will just be blue because it has no class and would take
+        the last color assigned to div tags in the styles.css file...which is
```

Comment:
There is a `<code>` element that works kind of like `<strong>` and you could use it here to denote that the content includes code (or in this case, a filename). 

```html
 "Scratching" will just be blue because it has no class and would take
the last color assigned to div tags in the <code>styles.css</code> file
```

Reply:
This is really helpful! I was wondering how to do that and didn't know about this tag 👍